CHANGELOG.md

```diff
@@ -0,0 +1,39 @@
+Change log
+===
+
+### Newer Versions 
+
+For newer versions. See [GitHub releases](https://github.com/jenkinsci/matlab-plugin/releases)
+
+### 1.0.3
+
+Release date: _Nov 05, 2019_
+
+* Bug Fix : Scratch file copy is restricted to "Automatic" option.
+* README doc updates. 
+
+
+### 1.0.2
+
+Release date: _Aug 21, 2019_
+
+* MATRIX build support. 
+* Plugin support for older versions of MATLAB till R2013a.

```

Comment:
I would use "until" in place of "till". It is more formal.

Reply:
Done.